packages/frontend/editor-ui/src/views/SigninView.vue

```diff
@@ -136,13 +134,6 @@ const login = async (form: LoginRequestDto) => {
 			mfaRecoveryCode: form.mfaRecoveryCode,
 		});
 		loading.value = false;
-		if (settingsStore.isCloudDeployment) {
-			try {
-				await cloudPlanStore.checkForCloudPlanData();
-			} catch (error) {
-				console.warn('Failed to check for cloud plan data', error);
-			}
-		}
```

Comment:
Moved logic to `init.ts`. The authenticated init function already gets called after the route changes after signin, rendering this unnecessary. Tested it with both login and logout. Works as expected.